src/OnlineSales/Program.cs

```diff
@@ -434,4 +450,58 @@ private static void ConfigureCORS(WebApplicationBuilder builder)
             });
         });
     }
+
+    private static void ConfigureIdentity(WebApplicationBuilder builder)
```

Comment:
@Exmirai consider to move ConfigureIdentity method away from Program.cs to avoid unlimited growth of the logic inside a single Program.cs file